network/dag/notifier.go

```diff
@@ -99,12 +102,14 @@ type NotifierOption func(notifier *notifier)
 // Event is the metadata that is stored for a notifier specific event
 // The Hash is used as identifier for the Event.
 type Event struct {
-	// Type of an event, can be used to filter
+	// Type of event, can be used to filter
 	Type string `json:"type,omitempty"`
 	// Hash is the ID of the Event, usually the same as the dag.Transaction.Ref()
 	Hash hash.SHA256Hash `json:"Hash"`
 	// Retries is the current number of retries
 	Retries int `json:"retries"`
+	// Latest records the timestamp of the last notification attempt. It is not used in the backoff calculation.
+	Latest *time.Time `json:"latest,omitempty"`
```

Comment:
Why can it be nil? An event is always attempted to be delivered at least once (initial publish)?

Makes for a harder interface

Reply:
`Save` method only saves the event without notification, so there is no latest attempt time yet

Reply:
also existing entries do not have it